modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -437,6 +437,18 @@ class NDB_Form_User_Accounts extends NDB_Form
         }
 
         $this->tpl_data['success'] = true;
+
+        if ($this->isCreatingNewUser()) {
+            $baseURL  = $config->getSetting('url');
+            $redirect = $baseURL
+                . "/"
+                . $this->name
```

Comment:
This doesn't seem to have any reason to use a variable, since it's always in the same module and it just adds a layer of indirection to figure out where it's going.